app/views/account/dashboard/_recent_searches.html.slim

```diff
@@ -0,0 +1,95 @@
+.card[style="height: 395.25px;"]
```

Comment:
fixed height? move to a css or better remove at all and use some lib for example https://github.com/liabru/jquery-match-height